### DISCUSSION TITLE
Pre-Release v15.0.0

### DISCUSSION THREAD

**Discussion by caco3** [jomjol/AI-on-the-edge-device#2065] [General] [answered]
We made a new Pre-release `v15.0.0`.

Since it modifies the `config.ini`, We would be glade for some testers.

**Make sure to create a backup beforehand!**

Release Notes: [v15.0.0](https://github.com/jomjol/AI-on-the-edge-device/releases/tag/v15.0.0)

**Comment by caco3** [comment]
@jomjol @Slider0007 @haverland @nliaudat Can you give it a try? If you where on rolling, please revert first to `v14.0.3` for a proper test of the migration!

**Reply by nliaudat** [reply]
Everything goes well in migrating config.ini.
The only thing I remarks is that OTA upload is very slow (1440b uploaded at each time), but it's related to 14.0.3 version

**Reply by ghost** [reply]
On my site updated from v14.0.3 without any issue. Update is a bit slow, upload was fast (few seconds).

Config looks good. 
Data Graph page is empty, but because it's disabled. Here could be a hint, but not really necessary for 15.0.0

**Reply by caco3** [reply, marked as answer]
Thanks for testing and the feedback!

AFAIK we did not do any changes on the firmware update. On my systems it runs as usual. Do you have a good wifi connection? The installation itself is also (slow) as usual, but we did not touch it either.

> Data Graph page is empty, but because it's disabled.

Thanks, that indeed was a bug. It is fixed now in https://github.com/jomjol/AI-on-the-edge-device/commit/3960823439ec5136bff730692bcc17cf9a8e4f0c resp. https://github.com/jomjol/AI-on-the-edge-device/commit/2c1a7f4c9e7fb676241306fdb82ea1b7adf0fbf8
I created a new release 15.0.1
